service/controller/v15/resource/bridgezone/resource.go

```diff
@@ -221,10 +220,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			},
 			HostedZoneId: &intermediateZoneID,
 		}
+		time.Sleep(1 * time.Second)
 		_, err := defaultGuest.ChangeResourceRecordSetsWithContext(ctx, in)
 		if err != nil {
 			return microerror.Mask(err)
 		}
+		time.Sleep(1 * time.Second)
```

Comment:
Because this resource is temporary and maybe exactly because of that reason not really thought through I just went with the lazy approach and added sleeps where necessary. My hope is that we work around the rate limit issues for now and just get over the postmortem. Later we want to remove this resource anyway.